.circleci/config.yml

```diff
@@ -111,18 +111,15 @@ jobs:
             - bin/
   release:
     environment:
-        CIRCLE_PROJECT_USERNAME: cloudskiff
-        CIRCLE_PROJECT_REPONAME: driftctl
+      CI: circleci
```

Comment:
You should use predefined vars instead of defining a custom one (that override defaults btw), scripts should be updated according.

https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables